### PR TITLE
Update docs for v8.11.0 release

### PR DESCRIPTION
**Pull request by mattc58**
This updates the release notes for `8.11.0 BC6`

### CODE REVIEW

**Comment by mattc58** [comment]
@elasticsearchmachine run elasticsearch-ci/docs

File: docs/reference/migration/migrate_8_11.asciidoc [hunk 1]
```diff
@@ -40,3 +40,32 @@ The `transport_versions` subobject of the response to `GET _cluster/state` has b
...
+
+[[rollup_functionality_deprecated]]
+.Rollup functionality is now deprecated
+[%collapsible]
+====
+*Details* +
+{ref}/xpack-rollup[Rollup functionality] has been deprecated and will be removed in a future release. Previously, rollups were available in technical preview.
+
+*Impact* +
+Use {ref}/downsampling.html[downsampling] to reduce storage costs for time series data by by storing it at reduced granularity.
```
Comment (abdonpijpelink) [inline docs/reference/migration/migrate_8_11.asciidoc:69]:
```suggestion
<<xpack-rollup,Rollup functionality>> has been deprecated and will be removed in a future release. Previously, rollups were available in technical preview.

*Impact* +
Use <<downsampling,downsampling>> to reduce storage costs for time series data by storing it at reduced granularity.
```
This should resolve the build error.

Comment (mattc58) [reply docs/reference/migration/migrate_8_11.asciidoc]: Thanks for doing this!

**Comment by arpadkiraly** [comment]
LGTM.
One thing to note: I believe we bump versions of docs only on release day, so you may want to hold back on merging this until then.

**Review by arpadkiraly — APPROVED** [review]

**Comment by mattc58** [comment]
Checked with Abdon and we will merge this now, and then there will be another one on release day or thereabouts.

**Pull request merged**